test/e2e/components/vl-map-search.js

```diff
@@ -0,0 +1,50 @@
+const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+
+class VlMapSearch extends VlElement {
+
+    async _getSearch() {
+        return this.shadowRoot;
+    }
+
+    async _getSelect() {
+        const search = await this._getSearch();
+        const select = await search.findElement(By.css('select[is="vl-select"]'));
+        return new VlSelect(this.driver, select);
+    }
+
+    async open() {
+        const select = await this._getSelect();
+        await select.open();
+    }
+
+    async search(text) {
+        const search = await this._getSearch();
+        const input = await search.findElement(By.css('.vl-select__list > input'));
+        await this.driver.executeScript(`arguments[0].value='${text}'`, input);
+        const select = await this._getSelect();
+        await this.driver.executeScript(`arguments[0].dispatchEvent(new CustomEvent('search', {detail: {value: '${text}'}}))`, select);
+    }
+
+    async selectByIndex(index) {
+        const select = await this._getSelect();
+        await select.selectByIndex(index);
+    }
+
+    async waitForValues() {
```

Comment:
Moet een gebruiker dit doen of zouden we dit niet zelf doen en verbergen voor de gebruiker?

Reply:
Op zich dacht ik van het bij de search te stoppen, maar als er geen values zijn is er een probleem... Misschien heeft iemand anders nog een beter idee?

Reply:
Als er geen adres gevonden wordt, toont hij een ander soort bericht ("Geen adres gevonden" of iets dergelijks).

Reply:
Ok ik zal dat ook proberen opvangen dan

Reply:
Niet direct gevonden, nieuwe taak gemaakt: https://www.milieuinfo.be/jira/browse/UIG-604